client/src/components/ChatTile.jsx

```diff
@@ -66,3 +61,1913 @@ function ChatTile (props) {
 }
 
 export default ChatTile;
+
+// provided by twitch
+const badges = {
```

Comment:
Is this something that could be moved to a JSON file so it doesn't clutter the JSX code? Genuine question, I'm not sure if JSON resource files are allowed in web dev.

Reply:
not easily